### ISSUE TITLE
suggested custom GPIO settings

### ISSUE THREAD

**Issue by rfeng74** [shrocky2/SiriGarage#10]
I would add ability to modify the GPIO setting assignments.  This is because some relay modules for Raspberry Pis are mapped differently and some can be stacked on top of the GPIO pins.  See attached pics.  Additionally the GPIO mappings for the Garage switches should be able to be mapped differently in a setting GUI.
![RPI_relay_stack1](https://user-images.githubusercontent.com/17171832/216752248-f8141ade-8628-4ef6-9b4b-de73bf2feb74.jpg)
![RPI_relay_stack2](https://user-images.githubusercontent.com/17171832/216752252-32412252-a710-4f2c-af4d-07dd288e64bb.jpg)

**Comment by rfeng74** [comment]
For example, the above relay board uses GPIO #s 37, 38 and 40 to toggle the relay.  In the code I had to map GPIO #37 for the door #3 open sensor to GPIO #35 so the relay board could use GPIO #37.  Since the relay board is hard wired to use GPIO #s 37, 38 and 40, I changed all instances of GPIO #7 to GPIO #37, GPIO #11 to GPIO #38 and GPIO #13 to GPIO #40 in all of the source code.  I am using a Raspberry Pi 2B.
